Code/CoreData/RKRelationshipConnectionOperation.m

```diff
@@ -50,8 +50,10 @@ static BOOL RKConnectionAttributeValuesIsNotConnectable(NSDictionary *attributeV
     NSMutableDictionary *destinationEntityAttributeValues = [NSMutableDictionary dictionaryWithCapacity:[connection.attributes count]];
     for (NSString *sourceAttribute in connection.attributes) {
         NSString *destinationAttribute = (connection.attributes)[sourceAttribute];
-        id sourceValue = [managedObject valueForKey:sourceAttribute];
-        [destinationEntityAttributeValues setValue:sourceValue ?: [NSNull null] forKey:destinationAttribute];
+        [managedObject.managedObjectContext performBlockAndWait:^{
```

Comment:
Good question, I don't really understand the differences in impact that would have. I guess having it outside the loop makes more sense.